Library/Formula/xcode-coveralls.rb

```diff
@@ -0,0 +1,14 @@
+class XcodeCoveralls < Formula
+  desc "A tool to upload code coverage from Xcode to coveralls.io."
+  homepage "https://macmade.github.io/xcode-coveralls"
+  url "https://github.com/macmade/xcode-coveralls/archive/0.1.0.tar.gz"
+  sha256 "17e671c39a2c037a40e5f35808744752c802d7884cd16f17bb247fa1bb337d59"
+  depends_on :xcode => 6.0
```

Comment:
Can you add an empty line in between "sha256" and the dependencies, and in between the dependencies and the install method?